multicast/__main__.py

```diff
@@ -523,75 +554,106 @@ def doStep(self, *args):
 
 
 def main(*argv):
-	"""Do main event stuff.
+	"""
+	Do main event stuff.
+
+	Executes the multicast command-line interface, by parsing command-line arguments and dispatching
+	the appropriate multicast operations.
 
 	The main(*args) function in multicast is expected to return a POSIX compatible exit code.
 	Regardless of errors the result as an 'exit code' (int) is returned.
 	The only exception is multicast.__main__.main(*args) which will exit with the underlying
 	return codes.
 	The expected return codes are as follows:
 		= 0:  Any nominal state (i.e. no errors and possibly success)
-		<=1:  Any erroneous state (caveat: includes simple failure)
+		=>1:  Any erroneous state (caveat: includes simple failure)
 		= 2:  Any failed state
 		= 3:  Any undefined (but assumed erroneous) state
-		> 0:  implicitly erroneous and treated same as abs(exit_code) would be.
+		=<0:  implicitly erroneous and treated same as abs(exit_code) would be.
```

Comment:
🙅🏻 the intent was to lineup the numbers and equals symbols for clearer readability. Your change is inferior visually for humans. Can we use a ≥ and ≤ for the >= and <= respectively, in the documentation. We can't use a space before as it will result in mixed spacing of tabs and spaces.

Reply:
I meant more like this:
```python3
		The expected return codes are as follows:
			= 0:  Any nominal state (i.e., no errors and possibly success)
			≥ 1:  Any erroneous state (includes simple failure)
			= 2:  Any failed state
			= 3:  Any undefined (but assumed erroneous) state
			≤ 0:  Implicitly erroneous and treated the same as abs(exit_code) would be.
```